test/scenario/broker/README.md

```diff
@@ -0,0 +1,29 @@
+abacus-broker-scenario-test
+===
+
+:information_source: In order to run the broker scenario test, export the following environment variables:
+
+```bash
+export CF_API_URI=https://api.<domain>
+export CF_USER=<prefix>abacus
```

Comment:
Just `<username>`. The `prefix` thing should be common sense.

Reply:
It is common sense, but I would like to be able to easily copy-paste and change the values in order to run the test locally, so I decided to leave it.